utils/database.js

```diff
@@ -195,7 +199,7 @@ const buildOrderByStmt = function(req, res, tableFields) {
     }
 };
   
-const buildSelectStmt = function(req, res, tableFields, sqlParams) {
+const buildSelectStmt = function(req, res, tableFields, sqlParams,next) {
```

Comment:
The next argument isn't used in the function.

Reply:
done, thanks.